free-courses-en.md

```diff
@@ -195,6 +195,7 @@
 * [Central Connecticut State University, Introduction to CS Using Java](http://chortle.ccsu.edu/CS151/cs151java.html)
 * [Introduction to Java](http://ocw.mit.edu/courses/electrical-engineering-and-computer-science/6-092-introduction-to-programming-in-java-january-iap-2010/)
 * [Java for Complete Beginners](http://courses.caveofprogramming.com/courses/java-for-complete-beginners)
+* [Java for Mobile Devices - Introducing Codename One](https://codenameone.teachable.com/p/java-for-mobile-devices-introducing-codename-one)
```

Comment:
Good addition. Thanks!

The f-p-b community has not discussed how much to allow courses with content that requires registration to unlock content. I imagine there would be a lot more engagement if the first segment at least was available without registration.

Reply:
Thanks!

I initially thought this can't be done after reading your comment and had that answer typed out. But I made a couple of changes that might solve this by making all the modules in that course into a "free preview".